software-templates/techdocs-template/skeleton/README.md

```diff
@@ -0,0 +1,63 @@
+[![Package Status](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml/badge.svg)](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml)
+
+
+# ${{values.name}}
+
+${{values.description}}
+
+## About
+
+This is a default [README.md](README.md) file generated by the template. Customize it for your project.
+
+Refer to the [index.md](docs/index.md) file for instructions on how to add documentation.
+
+
+## Setup
+
+Replace this section with instructions to set up your project locally. 
+### Prerequisites
+
+Use this section to list any prerequisites needed to install your project. Example:
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+List the steps to installs your project. For example:
+
+1. Clone the repo
+   ```sh
+   git clone https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}.git
+   ```
+1. Create a `config.local.yaml` file
+1. Set the `env` variable to `local` in the `config.local.yaml` file
+   ```yaml
+   env: local
+   ```
+
+### Usage
+
+Use this section to describe how to use your project. Example:
+
+```sh
+yarn start
+```
+
+## Getting Help or Reporting an Issue
+
+To report bugs/issues/features requests, please file an [issue]( https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/issues).
+
+
+## Contributing
+If you would like to contribute, please see our [contributing](CONTRIBUTING.md) guidelines.
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE-OF-CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## License
+
+Copyright 2023 Province of British Columbia
+
```

Comment:
I've hardcoded the year, which I don't feel good about. I didn't see a quick/simple way to inject the current year.  I'm going to investigate further to see if there is away to set this programmatically. If you know how, please let me know!

Reply:
Maybe something like: https://github.com/RoadieHQ/roadie-backstage-plugins/issues/962

Reply:
Also would help if I used the current year...

Reply:
We can add that to v2 or beyond - this is probably fine for now (good catch on the year)

Reply:
Ok, I will add a Jira ticket for this